src/mds/mdstypes.cc

```diff
@@ -622,8 +622,7 @@ void inode_load_vec_t::dump(Formatter *f)
 
 void inode_load_vec_t::generate_test_instances(list<inode_load_vec_t*>& ls)
 {
-  utime_t sample;
-  ls.push_back(new inode_load_vec_t(sample));
+  ls.push_back(new inode_load_vec_t(DecayRate()));
```

Comment:
default constructor of inode_load_vec_t is identical to inode_load_vec_t(DecayRate())

Reply:
Yes, I originally had deleted the default constructor because I wanted to always force the caller to specify the DecayRate. Unfortunately towards the end of developing this patch I didn't find an easy to make it work with the `src/test/encoding/...` encoder testing since it expects a default constructor. So I brought it back. In any case, I'm inclined to leave the above cases as-is.